cmd/soroban-cli/src/commands/contract/bindings/typescript.rs

```diff
@@ -134,18 +134,14 @@ impl NetworkRunnable for Cmd {
             .to_str()
             .ok_or_else(|| Error::NotUtf8(file_name.to_os_string()))?;
         print.infoln(format!("Embedding contract address: {contract_address}"));
-        p.init(
+        project.init(
             contract_name,
             &contract_address.to_string(),
             &network.rpc_url,
             &network.network_passphrase,
             &spec,
         )?;
         print.checkln("Generated!");
-        print.infoln(format!(
-            "Run \"npm install && npm run build\" in {:?} to build the JavaScript NPM package.",
-            self.output_dir
-        ));
```

Comment:
We just added this instruction because it is relevant to the typescript bindings, which create an npm package, but the cli no longer installs and builds the package. That is now the job of the user. This command is independent of the frontend template feature that used to exist in the contract init command.

Reply:
this instruction won't work unless you follow whatever we had on the template; `npm run build` is not a standard task and can be something completely different depending on how you set up your project.

Let's say you have a hello smart contract project, and the dapp lives somewhere else, and you're using `stellar contract bindings typescript --out-dir ../hello-dapp`, then this also is misleading, because I'd run it on the smart contract dir.

Reply:
oh, i just got the message. It doesn't mention the current dir, it mentions the output dir, which has everything mentioned on the task. My bad! 🤦‍♂️